package.json

```diff
@@ -62,8 +62,6 @@
     "karma-sourcemap-loader": "^0.3.6",
     "karma-spec-reporter": "0.0.23",
     "karma-webpack": "^1.7.0",
-    "lodash.debounce": "^4.0.3",
-    "lodash.throttle": "^4.0.1",
```

Comment:
I removed the throttling on the store listener, since it now doesn't call model.update except when the selected mark changes there is very little logic that would run each digest so the impact is minimal. These tools got us through earlier hurdles but the value of these methods has finally outweighed their cost by obscuring the model state sync issue.